packages/google_sign_in/README.md

```diff
@@ -0,0 +1,84 @@
+# google_sign_in_tizen
+
+[![pub package](https://img.shields.io/pub/v/google_sign_in_tizen.svg)](https://pub.dev/packages/google_sign_in_tizen)
+
+The Tizen implementation of [`google_sign_in`](https://github.com/flutter/plugins/tree/master_archive/packages/google_sign_in/google_sign_in).
+
+## Usage
+
+This package is not an _endorsed_ implementation of `google_sign_in`. Therefore, you have to include `google_sign_in_tizen` alongside `google_sign_in` as dependencies in your `pubspec.yaml` file.
+
+```yaml
+dependencies:
+  google_sign_in: ^5.4.0
+  google_sign_in_tizen: ^0.1.0
+```
+
+For detailed usage on how to use `google_sign_in`, see https://pub.dev/packages/google_sign_in#usage.
+
+## Tizen integration
+
+`google_sign_in_tizen` supports Google Sign-In for [TVs and Limited Input Devices](https://developers.google.com/identity/gsi/web/guides/devices), note that many API scopes are limited for these device types, see [supported scopes](https://developers.google.com/identity/protocols/oauth2/limited-input-device#allowedscopes).
+
+The plugin uses [OAuth 2.0 Device Authorization Grant](https://datatracker.ietf.org/doc/html/rfc8628) (aka device flow) which has a different authorization flow than [Authorization Code Grant with PKCE](https://datatracker.ietf.org/doc/html/rfc7636) used for endorsed platforms (Android, iOS, and web). While every step needed to authorize the app for "authorization code grant with PKCE" happens on a single device, "device flow" requires the user to use a secondary device (PC, tablet, or desktop) to finalize the authorization.
+
+To access Google Sign-In for Tizen, you'll need to make sure to [register your application for TVs and limited input devices](https://developers.google.com/identity/gsi/web/guides/devices). You also need to add some additional code to your app to fully integrate `google_sign_in` on Tizen (explained below).
+
+### Adding OAuth credentials
+
+Unlike "authorization code grant with PKCE", "device flow" requires [client secret](https://developers.google.com/identity/protocols/oauth2/limited-input-device#step-4:-poll-googles-authorization-server) parameter during token request. You must call `setCredentials` function with your client's OAuth credentials before calling `google_sign_in`'s API.
```

Comment:
- Shouldn't "authorization code grant with PKCE" and "device flow" be capitalized?
- "a" client secret parameter
- You're assuming the reader already knows what "during token request" means. Could you reword if it's not the case?

Reply:
> Shouldn't "authorization code grant with PKCE" and "device flow" be capitalized?

Yes they should.

> You're assuming the reader already knows what "during token request" means. Could you reword if it's not the case?

How about:

Unlike "Authorization Code Grant with PKCE", "Device Flow" requires a [client secret](https://developers.google.com/identity/protocols/oauth2/limited-input-device#step-4:-poll-googles-authorization-server) parameter during Google Sign-In.

Reply:
Looks good. Be sure to be consistent throughout the document.